spring-batch-2/pom.xml

```diff
@@ -70,9 +70,46 @@
             <artifactId>spring-core</artifactId>
             <version>${spring-core.version}</version>
         </dependency>
```

Comment:
jakarta.xml.bind-api, httpclient and jettison
aren't required in spring-batch pom, let's check.